rmf_fleet_adapter/src/rmf_fleet_adapter/agv/FleetUpdateHandle.cpp

```diff
@@ -2538,6 +2538,13 @@ FleetUpdateHandle& FleetUpdateHandle::set_retreat_to_charger_interval(
   return *this;
 }
 
+//==============================================================================
+FleetUpdateHandle& FleetUpdateHandle::reassign_dispatched_tasks()
+{
+  _pimpl->reassign_dispatched_tasks([]() {}, [](auto) {});
```

Comment:
This needs to be scheduled inside of the fleet handle's worker, similar to what's done for the [RobotUpdateHandle version](https://github.com/open-rmf/rmf_ros2/blob/132e516d5c7e55486cd30ba422ee09a1e648fc1e/rmf_fleet_adapter/src/rmf_fleet_adapter/agv/RobotUpdateHandle.cpp#L807-L813) of this function. Neglecting to do that can lead to segmentation faults and undefined behavior.